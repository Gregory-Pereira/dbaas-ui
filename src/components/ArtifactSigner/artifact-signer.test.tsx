import React from 'react';
import { render, screen } from '@testing-library/react';

test('expect sample-component to render children', () => {
  const { container } = render(<div>Sample Component</div>);
  expect(container).toMatchSnapshot();
  expect(screen.getByText('Sample Component')).toBeInTheDocument();
});

export {};
