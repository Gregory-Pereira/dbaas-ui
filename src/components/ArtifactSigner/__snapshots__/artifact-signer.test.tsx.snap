// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect sample-component to render children 1`] = `
<div>
  <div>
    Sample Component
  </div>
</div>
`;
